README.md

```diff
@@ -1,2 +1,2 @@
-# packer-base-rstudio
+# packer-base-shinyserver
```

Comment:
I recommend copying some of the boilerplate readme info from [packer-ami-template README](https://github.com/Sage-Bionetworks-IT/packer-ami-template/blob/master/README.md)

Reply:
Yeah, I read the Confluence page after the fact...